test_data/yam/agama_sle.yaml

```diff
@@ -1,2 +1,2 @@
 ---
-os_release_name: SLE
+os_release_name: SLES
```

Comment:
we only needs this change in this PR:
https://openqa.suse.de/tests/15770497#step/validate_product/1

Reply:
Done